trade-finance-manager-api/src/constants/email-template-ids.js

```diff
@@ -12,6 +12,13 @@ const GEF_DEAL_SUBMIT_CONFIRMATION = 'b6c8ed30-dcef-4dc5-afbf-5cafd62c5a5b';
 const TFM_FEEDBACK_RECEIVED = '967b3706-8ea8-4cea-a315-1436cd4073f8';
 const MIGRATED_DEAL_SUBMITTED = 'c709f2e9-e1c7-4de9-9a46-a7c0726972a0';
 const AUTOMATIC_AMENDMENT = 'e1c1be85-b497-4fdd-9988-6501839456fc';
+const MANUAL_AMENDMENT_DECISION_APPROVED_W_CONDITIONS = '96ef05ce-edb3-4ca8-998d-b2d8261431bc';
```

Comment:
are we not combining some of the email templates?

Reply:
not yet, is being 'discussed'